services/worker-manager/src/main.js

```diff
@@ -189,10 +176,10 @@ let load = loader({
   },
 
   providers: {
-    requires: ['cfg', 'monitor', 'notify', 'db', 'estimator', 'WorkerPoolError', 'schemaset'],
-    setup: async ({cfg, monitor, notify, db, estimator, WorkerPoolError, schemaset}) =>
+    requires: ['cfg', 'monitor', 'notify', 'db', 'estimator', 'schemaset'],
+    setup: async ({cfg, monitor, notify, db, estimator, schemaset}) =>
       new Providers().setup({
-        cfg, monitor, notify, db, estimator, WorkerPoolError,
+        cfg, monitor, notify, db, estimator, Worker, WorkerPoolError,
```

Comment:
Did you mean to remove `WorkerPoolError` rather than add `Worker`?

Reply:
Removed both of these here.